core/node/house_keeper/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "zksync_house_keeper"
+version = "0.1.0"
+edition.workspace = true
+authors.workspace = true
+homepage.workspace = true
+repository.workspace = true
+license.workspace = true
+keywords.workspace = true
+categories.workspace = true
+
+[dependencies]
+vise.workspace = true
+zksync_dal.workspace = true
+zksync_shared_metrics.workspace = true
+zksync_utils.workspace = true
```

Comment:
Nit: Looks like it's only used for `seconds_since_epoch` in a single place; IMO, it could make sense to inline this call.

Reply:
do u want to say that if there is a single call place we have to inline?